Sources/eppo/EppoClient.swift

```diff
@@ -31,7 +31,6 @@ public class EppoClient {
 
     private static let sharedLock = NSLock()
     private static var sharedInstance: EppoClient?
-    private static let initializerQueue = DispatchQueue(label: "cloud.eppo.client.initializer")
```

Comment:
I could be wrong, but going off this SO explanation

https://stackoverflow.com/a/78458487

It sounds like `DispatchQueue.async` and `Task` is mixing old (GCD) and new (structured concurrency) approaches, and we don't need both and should use the modern system exclusively. Open to discussion if anyone feels we should continue to use `DispatchQueue` here

Reply:
AI agrees 📈 

![image](https://github.com/user-attachments/assets/c780d573-18bf-4239-ae11-aab39379807a)

Reply:
Counterpoint: this comment suggests removing DispatchQueue usage could cause issues with old iOS versions

https://github.com/Eppo-exp/eppo-ios-sdk/blob/20acd299dda1a63fddfdeebea6305d96bead6571/Sources/eppo/ConfigurationStore.swift#L4-L8

However, I didn't add any new `async` or `await` to the code. The `await` in the `Task` was already present

Reply:
Just don't use `DispatchQueue.asyncOrWait` - see the issue in the comment for more details.